### DISCUSSION TITLE
Unable to submit fastMRI reconstruction result

### DISCUSSION THREAD

**Discussion by yuezewang** [facebookresearch/fastMRI#249] [Q&A] [answered]
Hello, I can't submit the reconstruction result, because it keeps getting stuck.  ("Processing, please wait...")

**Comment by mmuckley** [comment, marked as answer]
Hello @yuezewang, I don't see any stuck submissions at the moment. Could you provide a little more detail?

**Reply by yuezewang** [reply]
Although it was stuck for a long time, the result was successfully returned in the end. Thanks for your attention! ^ ^
